acitoolkit/acisession.py

```diff
@@ -137,6 +137,7 @@ def run(self):
             except ConnectionError:
                 logging.error('Could not relogin to APIC due to ConnectionError')
                 self._apic.login_error = True
+                self._exit = True
```

Comment:
nit: since there is a public method for this, we should probably just call that.

Reply:
Hmm I think public method is more for the outside users of this class. So you can see that in this Login() class, they all try to access the _exit member variable directly. One example is listed here,
https://github.com/noironetworks/acitoolkit/blob/noiro-lite/acitoolkit/acisession.py#L118

Reply:
In general I like to keep encapsulation in the method -- particularly if the method does something other than set the value of a member (or might do something different in the future). In this case, it's only setting the variable, and probably will only ever do that, so it's probably okay.